src/routes/charts.js

```diff
@@ -669,14 +669,18 @@ async function editChart(request, h) {
 async function exportChart(request, h) {
     const { payload, params, auth, logger, server } = request;
     const { events, event } = server.app;
+    const user = auth.artifacts;
 
-    if (auth.artifacts.role === 'guest') return Boom.forbidden();
+    const userPlugins = await user.getUserPluginCache();
+    if (params.format !== 'png' && !(userPlugins && userPlugins.plugins.includes('export-pdf'))) {
```

Comment:
this test would also be true if a fictional plugin named `prevent-export-pdf` would be assigned to a user. the `plugins` field is a string and has to be split by `,`.

```suggestion
    if (params.format !== 'png' && !(userPlugins && userPlugins.plugins.split(',').includes('export-pdf'))) {
```